crates/sui-types/src/execution_config_utils.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_binary_format::binary_config::{BinaryConfig, TableConfig};
+use sui_protocol_config::ProtocolConfig;
+
+/// Build a `BinaryConfig` from a `ProtocolConfig`
+pub fn to_binary_config(protocol_config: &ProtocolConfig) -> BinaryConfig {
+    BinaryConfig::new(
+        protocol_config.move_binary_format_version(),
+        protocol_config.no_extraneous_module_bytes(),
+        TableConfig {
```

Comment:
Is u16::MAX the right default here? It seems rather large...

Reply:
mostly what used to be